widgets/ScrollTable.lua

```diff
@@ -468,6 +468,9 @@ local methods = {
 			elseif format == 'moneyShort' then
 				value = table.stdUi.Util.formatMoney(value, true);
 				cellFrame.text:SetText(value);
+			elseif format == 'formatTime' then
```

Comment:
well, for consistency, this should be a `'time'` format

Reply:
Done!